learn.py

```diff
@@ -0,0 +1,34 @@
+import openpyxl.workbook
+import pandas as pd
+import numpy  as np
+import openpyxl
+from openpyxl.styles import PatternFill
+
+rows, cols = 100, 100
+data=np.random.randint(-100,100,size=(rows,cols))
+df=pd.DataFrame(data)
+df.to_excel('random_data.xlsx',index=False)
+
+df['Row_mean'] = df.mean(axis=1)
+df.loc['Col_mean'] = df.mean(axis=0)
```

Comment:
axis là gì? tạo sao khác nhâu? comment giải thích

Reply:
axis là trục được chỉ định để thao tác tính toán
axis=1 tính toán được thực hiện theo hàng
axis=0 tính toán được thực hiện theo cột